src/en/help-google.md

```diff
@@ -0,0 +1,114 @@
+Title: Help with Google Compute Engine clouds
+
+# Using the Google Compute Engine public cloud
+
+Juju already has knowledge of the GCE cloud, so unlike previous versions there
+is no need to provide a specific configuration for it, it 'just works'. GCE
+will appear in the list of known clouds when you issue the command:
+  
+```bash
+juju list-clouds
+```
+And you can see more specific information (e.g. the supported regions) by 
+running:
+  
+```bash
+juju show-cloud google
+```
+
+If at any point you believe Juju's information is out of date (e.g. Amazon just 
```

Comment:
I'm not that bothered about it. Really, the *only* thing that is ever likely to happen is that someone adds a new region. Juju isn't the mechanism people will use to know that anyhow.  So the only time anyone actually *needs* to update their cloud is when they _know_ a new region has been added and they want to use it.
I originally thought that it would be a good idea for people to run this regularly, but really, I don't think it is.